### PR TITLE
env_mirror: fix a few leaks

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: include/rocksdb/utilities/env_mirror.h [hunk 1]
```diff
@@ -33,9 +33,21 @@ class WritableFileMirror;
 
 class EnvMirror : public EnvWrapper {
   Env* a_, *b_;
+  bool free_a_, free_b_;
```
Comment (IslamAbdelRahman) [inline include/rocksdb/utilities/env_mirror.h]: Where are we going to use these options ? 
I don't see we are calling them anywhere

Comment (liewegas) [inline include/rocksdb/utilities/env_mirror.h]: There are no in-tree users of EnvMirror.  It's a debugging tool for those implementing new Env's.  Ceph uses it here: https://github.com/ceph/ceph/blob/master/src/os/bluestore/BlueStore.cc#L3044-L3061

**Comment by liewegas** [comment]
@IslamAbdelRahman ping

**Comment by IslamAbdelRahman** [comment]
Thanks @liewegas

**Pull request merged**